adapta/storage/blob/base.py

```diff
@@ -151,6 +162,17 @@ def copy_blob(self, blob_path: DataPath, target_blob_path: DataPath, doze_period
         :param doze_period_ms: number of ms to doze between polling the status of the copy.
         """
 
+    @abstractmethod
+    def upload_data(self, source_file_path: str, target_file_path: DataPath, doze_period_ms: int) -> None:
```

Comment:
upload_blobs is better name imo

Reply:
I agree, it has been change in all places with the commit  [3943e2d](https://github.com/SneaksAndData/adapta/pull/437/commits/3943e2dda0291afba169f072092af8a0ff6741b7)

Thank you for the suggestion 🙂